src/libmongoc/src/mongoc/mongoc-write-command.c

```diff
@@ -989,15 +989,16 @@ _set_error_from_response (bson_t *bson_array,
 {
    bson_iter_t array_iter;
    bson_iter_t doc_iter;
-   mcommon_string_t *compound_err;
    const char *errmsg = NULL;
    int32_t code = 0;
    uint32_t n_keys, i;
 
-   compound_err = mcommon_string_new (NULL);
+   mcommon_string_append_t compound_err;
+   mcommon_string_append_new_fixed_capacity (&compound_err, sizeof error->message - 1);
```

Comment:
```suggestion
   mcommon_string_append_new_fixed_capacity (&compound_err, sizeof (error->message) - 1u);
```

Reply:
What's the reason for the extra parens here?